### PR TITLE
Finishing

### PR DESCRIPTION
**Pull request by Shilpi75**

### CODE REVIEW

File: app/src/main/java/app/tasknearby/yashcreations/com/tasknearby/DetailActivity.java [hunk 1]
```diff
@@ -38,6 +38,7 @@
      * Views.
      */
     private Button doneButton;
+    private ActionBar mActionBar;
```
Comment (vermayash8) [inline app/src/main/java/app/tasknearby/yashcreations/com/tasknearby/DetailActivity.java]: why is this needed as a class member?

File: app/src/main/java/app/tasknearby/yashcreations/com/tasknearby/utils/AppUtils.java [hunk 2]
```diff
@@ -56,6 +56,11 @@ public static String getReadableDate(@NonNull Context context, @Nullable Date da
         }
     }
 
+    // TODO: Complete this
```
Comment (vermayash8) [inline app/src/main/java/app/tasknearby/yashcreations/com/tasknearby/utils/AppUtils.java]: Why is this method left blank? Compare like we're doing above this and fill this. DateOnlyInstance is not needed, you can compare by `date1.compare(new LocalDate()) == 0`.

Comment (Shilpi75) [reply app/src/main/java/app/tasknearby/yashcreations/com/tasknearby/utils/AppUtils.java]: yes! I'll. left that for now.

File: app/src/main/java/app/tasknearby/yashcreations/com/tasknearby/DetailActivity.java (hunk 1 above)
Comment (Shilpi75) [reply app/src/main/java/app/tasknearby/yashcreations/com/tasknearby/DetailActivity.java]: Maybe it appeared while resolving conflicts.
